code/datums/components/defibrillator.dm

```diff
@@ -308,7 +309,8 @@
 		target.Paralyse(10 SECONDS)
 		target.emote("gasp")
 
-		if(target.getBrainLoss() >= 100)
+		// Check if the brain has more than a critical amount of brain damage
+		if(sponge.damage >= (sponge.max_damage * BRAIN_DAMAGE_RATIO_CRITICAL))
```

Comment:
I dont like, it could lead to contributors fucking up brain damage, I think this should be a new proc for brain damage ratios. Something like `if(target.isBrainDamaged(BRAIN_DAMAGE_RATIO_CRITICAL))` would be much better.